pydantic/main.py

```diff
@@ -242,6 +242,11 @@ def __new__(mcs, name, bases, namespace, **kwargs):  # noqa C901
 
         prepare_config(config, name)
 
+        untouched_types = UNTOUCHED_TYPES
```

Comment:
So it doesn't take into account config.untouched_types anymore?

Reply:
it does because `untouched_types` is changed on line 253. I think this should keep the existing functionality except for cython behaviour.

But to be honest this is going to conflict a lot with #2094 anyway. Hopefully I can wait for that to be merged, then I'll amend this PR.

Happy new year! Thanks so much for all your support on pydantic in 2020.

If we're ever at the same conference (once conferences happen again :-() we should make an effort to meet in person.

Reply:
Ah right I went too fast over the patch and didn't read the whole file sorry
I'll work on #2094 tomorrow. Thanks for your kind message